### PR TITLE
Update `membrane_core` to 0.10.0. Fix tests

### PR DESCRIPTION
**Pull request by mickel8**
https://www.diffchecker.com/k3dxda4J

### CODE REVIEW

**Comment by mickel8** [comment]
I pasted in the description, difference between old reference file used in `"depayloading rtp with vp8" ` test (on the left in the diff) and current output of this test (on the right in the diff). When running `ffplay test/results/result.ivf` everything looks fine. I suppose that those small changes in timestamps are related to getting rid of `Ratio.t()` in some other repositories but I am not 100% sure.

**Review by varsill — CHANGES_REQUESTED** [review]

File: mix.exs [hunk 1]
```diff
@@ -37,14 +37,17 @@ defmodule Membrane.RTP.VP8.Plugin.Mixfile do
...
-      {:membrane_rtp_format, "~> 0.3.1"},
-      {:membrane_vp8_format, "~> 0.3.0"},
-      {:membrane_file_plugin, "~> 0.7.0", only: :test},
-      {:membrane_rtp_plugin, "~> 0.9.0", only: :test},
+      {:membrane_core, "~> 0.10.0"},
+      {:membrane_rtp_format, "~> 0.4.0"},
+      {:membrane_vp8_format, "~> 0.4.0"},
+      {:membrane_file_plugin, "~> 0.12.0"},
+      {:membrane_rtp_plugin,
+       github: "membraneframework/membrane_rtp_plugin", branch: "fix-ssrcrouter-eos", only: :test},
```
[the remark this answers is not in the document]
Comment (mickel8) [reply mix.exs:45]: Yeap, exactly, I am waiting for RTP plugin

File: test/test_helper.exs [hunk 2]
```diff
@@ -1 +1 @@
-ExUnit.start()
+ExUnit.start(capture_log: true)
```
[the remark this answers is not in the document]
Comment (mickel8) [reply test/test_helper.exs:1]: Thanks to this we will see debug logs only when some test fails. I think we started addding this in other repos too

[the remark this answers is not in the document]
Comment (varsill) [reply test/test_helper.exs:1]: Oh, that's a good idea.

**Review by mat-hek — APPROVED** [review]

**Pull request merged**